src/xo-app/logs/index.js

```diff
@@ -93,27 +93,38 @@ const JobTransferredDataInfos = ({ start, end, size }) => <div>
 </div>
 
 const CALL_FILTER_OPTIONS = [
-  {label: _('successfulJobCall'), value: 'success'},
-  {label: _('failedJobCall'), value: 'failed'},
-  {label: _('jobCallInProgess'), value: 'busy'},
-  {label: _('allJobCalls'), value: 'all'}
+  {label: 'successfulJobCall', value: 'success'},
+  {label: 'failedJobCall', value: 'error'},
+  {label: 'jobCallInProgess', value: 'running'},
+  {label: 'allJobCalls', value: 'all'}
 ]
 
+const PREDICATES = {
+  all: () => true,
+  error: call => call.error !== undefined,
+  running: call => call.end === undefined && call.error === undefined,
+  success: call => call.end !== undefined && call.error === undefined
+}
+
 class Log extends BaseComponent {
   state = {
     filter: 'all'
   }
 
   render () {
     const { props, state } = this
+    const predicate = PREDICATES[state.filter]
 
     return <div>
       <select
         className='form-control'
         onChange={this.linkState('filter')}
         value={state.filter}
       >
-        {map(CALL_FILTER_OPTIONS, ({ label, value }) => <option key={value} value={value}>{label}</option>)}
+        {map(CALL_FILTER_OPTIONS, ({ label, value }) => _(
+          label,
+          message => <option key={value} value={value}>{message}</option>
```

Comment:
`key` is a special prop: it's a property for the parent and not the child.

https://github.com/vatesfr/xo-web/blob/bb4ebcd198a4939a54142163c99725b7905c94cd/src/xo-app/settings/remotes/index.js#L348